### DISCUSSION TITLE
有没有和我一样是女王大人死忠粉的

### DISCUSSION THREAD

**Discussion by RAN124470** [share121/inter-knot#2582] [其他]
![1671766104339_看图王](https://github.com/user-attachments/assets/a4f5f4e1-3754-4eb8-9650-7f92d8481d7b)
女王大人万岁！
万岁！
万万岁！

**Comment by hyperion-netizen** [comment]
女王萬歲 不絕於耳!

**Reply by RAN124470** [reply]
不绝于耳
